A diagnostic test has a 98% probability of giving a positive result when applied to a person suffering from Thripshaw's Disease, and 10% probability of giving a (false) positive when applied to a non-sufferer. It is estimated that 0.5 % of the population are sufferers. Suppose that the test is now administered to a person whose disease status is unknown. <br/>Calculate the probability that the test will: <br/>

Be positive <br/>
Correctly diagnose a sufferer of Thripshaw's <br/>
Correctly identify a non-sufferer of Thripshaw's <br/>
Misclassify the person <br/>

In [3]:
#Probability of being positive
p = (.98*.005)+(.1*(1-.005)) #98% of the known population of sufferers and 10% of the rest of the population
print(p)

0.1044


<i></u>Correctly diagnose a sufferer of Thripshaw's </u></i><br/>
98% <br/><br/>
That answer was given in the problem description

<i></u>Correctly identify a non-sufferer of Thripshaw's </u></i><br/>
90% <br/><br/>
Only ten percent of non-sufferer's receive a false-positve. The remaining receive the correct result.

In [10]:
#Misclassify a person
pm = 1 -((.98*.005) + (.9*(1-.005))) #98% known population and 90% of the non-sufferers who are correctly tested
print(round(pm, 4))

0.0996


<u>Answers:</u> <br/>
Be positive <b>10.44%</b><br/>
Correctly diagnose a sufferer of Thripshaw's <b>98%</b><br/>
Correctly identify a non-sufferer of Thripshaw's <b>90%</b><br/>
Misclassify the person <b>9.96%<b/>